src/concrete/ml/torch/hybrid_model.py

```diff
@@ -612,7 +630,7 @@ def compile_model(
                     vals = self.private_q_modules[name].quant_layers_dict.values()
                     _, q_op = next(iter(vals))
                     const_inp = q_op.constant_inputs[1]  # Get the weights, the bias is in [2]
-                    const_inp.values = const_inp.qvalues.astype(numpy.float32)
+                    # const_inp.values = const_inp.qvalues.astype(numpy.float32)
```

Comment:
should't we allow this optimization if the quantization is static ?

Reply:
Ok I add it back for static quantization. Not sure the gain though.